experimental/ni_coq/oak-ni-coq.opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+maintainer: "aferr@google.com"
+authors:"Google LLC and contributors"
+synopsis: "Non-interference proofs for Project Oak"
```

Comment:
Nit: noninterference (with no hyphen) is the more common spelling in the literature. (I realize the wikipedia article does hyphenate it). The [original paper](https://www.cs.purdue.edu/homes/ninghui/readings/AccessControl/goguen_meseguer_82.pdf) also does not spell it with a hyphen.